src/appshell/qml/AboutDialog.qml

```diff
@@ -94,42 +94,45 @@ StyledDialogView {
             StyledTextLabel {
                 Layout.fillWidth: true
                 text: {
-                    var message = qsTrc("appshell", "Visit %1 for new versions and more information.\nGet %2 with the program or %3 to its development.")
-                    var museScoreUrl = aboutModel.museScoreUrl()
-                    var museScoreForumUrl = aboutModel.museScoreForumUrl()
-                    var museScoreContributionUrl = aboutModel.museScoreContributionUrl()
-                    return message
-                    .arg("<a href='" + museScoreUrl.url + "'>" + museScoreUrl.displayName + "</a>")
-                    .arg("<a href='" + museScoreForumUrl.url + "'>" + museScoreForumUrl.displayName + "</a>")
-                    .arg("<a href='" + museScoreContributionUrl.url + "'>" + museScoreContributionUrl.displayName + "</a>")
-                    .replace("\n", "<br>")
+                    let museScoreUrl = aboutModel.museScoreUrl()
+
+                    //: %1 will be a link to the MuseScore website
+                    let line1 = qsTrc("appshell", "Visit %1 for new versions and more information.")
```

Comment:
Maybe set the context to `appshell/about`, as we discussed?

Reply:
Done!